### DISCUSSION TITLE
how to start several bots but with customised scripts for each one

### DISCUSSION THREAD

**Discussion by RyterX** [ORelio/Minecraft-Console-Client#2493] [Q&A] [answered]
Hello, i would create 3 bot: 1 for farming wheat, 1 carrot and last for potato. How i can create a "specific" script for a certain bot?
so: bot1 execute scripts "potato"
bot2 "carrot" ...

I need to run a script because after relog i have to re-send the farmer command, so with scripts i can automate them to start farming at every relog

**Comment by milutinke** [comment, marked as answer]
You need to use 3 different accounts and 3 different configuration files.
You can use a different configuration file by passing it's name as the first parameter in the command for starting the client.
Eg. `./MinecraftClient MinecraftClient1.ini`
As for starting the farmer chat bot after reconnecting, use the Task Scheduler with task that fires on login.
